groups/ecs/ecs.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_ecs_cluster" "grafana_cluster" {
+  name = "${var.service}-cluster"
+}
+
+resource "aws_ecs_task_definition" "grafana_task" {
+  family                = "${var.service}-task"
+  network_mode          = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu = "256"
+  memory = "512"
+
+  execution_role_arn = aws_iam_role.ecs_execution_role.arn
+  task_role_arn = aws_iam_role.ecs_task_role.arn
+
+  container_definitions = <<DEFINITION
+  [
+    {
+      "name": "grafana-container",
+      "image": "${local.image_owner_id}.dkr.ecr.${var.region}.amazonaws.com/${var.image_repository_name}:${var.grafana_image_version}",
+      "essential": true,
```

Comment:
This will lead to invalid specification depend on which account this code is executed against. 

The ECR repository URI should be specificed wholly, as a variable, and not compounded like this as there's only one ECR repository URI for the container image.

Also you also defined `local.grafana_image` for this and didn't use it.